app/routes.js

```diff
@@ -1,7 +1,8 @@
-// const controller = require('./controllers/controller');
+const users = require('./controllers/users.js'),
+  emailValidator = require('./middlewares/emailValidator').handle,
```

Comment:
This is not bad, but what happens if exist 30 validations? Exist packages for validate this things, for example `express-validator-check` . Investigate this and change the logic for middleware